lib/ui/focusutils.js

```diff
@@ -50,48 +44,42 @@ class FocusHistory {
 }
 
 export function activate (i) {
-  ink = i
-  let history = new FocusHistory(30)
+  subs.add(atom.workspace.onDidStopChangingActivePaneItem((item) => {
+    if (item instanceof TextEditor) {
+      lastEditor = item
+    } else if (item instanceof i.InkTerminal) {
+      lastTerminal = item
+    }
+  }))
 
+  const ink = i
```

Comment:
Don't need this anymore -- should just change the `i` function argument to `ink` and that's it.

Reply:
Ah I missed to refactor it. Fixed !